db/seeds.rb

```diff
@@ -80,19 +81,29 @@
 
 FactoryBot.create(:vacancy,
                   job_title: 'PE Teacher',
-                  subject: Subject.last,
-                  school: ealing_school,
+                  subject: Subject.find_by!(name: 'Physical Education'),
+                  school: academy,
                   working_patterns: ['part_time'],
                   minimum_salary: 30000,
                   maximum_salary: 35000,
                   min_pay_scale: payscale,
                   leadership: leadership)
 
+FactoryBot.create(:vacancy,
```

Comment:
Nice. I did wonder if we could make use of the `seed_from_api` task when seeding in dev. Probably something for later though